client/flags/flags.go

```diff
@@ -77,6 +77,7 @@ const (
 	FlagTip              = "tip"
 	FlagAux              = "aux"
 	FlagOutput           = tmcli.OutputFlag
+	FlagGenesisFilePath  = "genesis-path"
```

Comment:
Is there an use case where we would want the genesis to be exported somewhere else than the app directory? It seems like a logic place for me to use the app home. That simplify quite some logic too.

Reply:
Sure, I thought the node operator might like to export the states to other places. It can be removed to simplify.